packages/create-hintrc/package.json

```diff
@@ -10,8 +10,10 @@
     "create-hintrc": "./dist/src/create-hintrc.js"
   },
   "dependencies": {
+    "@hint/utils": "^1.0.0",
     "browserslist": "^4.5.4",
-    "encoding": "^0.1.12"
+    "encoding": "^0.1.12",
```

Comment:
Why do we need `encoding` in this package?

Reply:
I don't know xD

Reply:
We are just telling webpack how it should be loaded, can't seem to find where it's being used 🤔